### ISSUE TITLE
data(met) doc should use a newer link

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1186]
Not that I have any expectation of finding one. Jeeze, the envcan website is terrible. You can find data but not linkable data, i.e. you have to click a button, which I guess makes java magic happen, to download a file from the browser. That's pißß useless (see what I did there?) for downloading data within a program. And, damned if I can even find the data that I have in `data(met)` ... seems like I can find that day but the pressures don't match.  Below is a link where I was looking (not exactly a clean link!).  If I cannot find a way to download data from envcan (or whatever one calls weather.gc.ca) I could be convinced to simply retire the `met` object type from oce ... what's the point in supporting a few data files I have on my machine, when it seems impossible to figure out how to get other data files? 

http://climate.weather.gc.ca/climate_data/hourly_data_e.html?hlyRange=1953-01-01%7C2017-01-31&dlyRange=1944-02-01%7C2007-12-12&mlyRange=1944-01-01%7C2007-03-01&StationID=6465&Prov=NS&urlExtension=_e.html&searchType=stnProx&optLimit=specDate&Month=9&Day=30&StartYear=1840&EndYear=2017&Year=2003&selRowPerPage=25&Line=2&txtRadius=25&optProxType=city&selCity=44%7C40%7C63%7C36%7CHalifax&selPark=&txtCentralLatDeg=&txtCentralLatMin=0&txtCentralLatSec=0&txtCentralLongDeg=&txtCentralLongMin=0&txtCentralLongSec=0&timeframe=1

**Comment by dankelley** [comment]
I found the data but the only way to get it is to click the "go" button on the following link. Also, when I look at the dataset, compared with the previous one (snapshot) I see that they are expressing the temperature unit differently (cv #1184)

http://climate.weather.gc.ca/climate_data/hourly_data_e.html?hlyRange=1961-01-01%7C2012-09-13&dlyRange=1953-01-01%7C2012-09-13&mlyRange=1953-01-01%7C2012-09-01&StationID=6358&Prov=NS&urlExtension=_e.html&searchType=stnProv&optLimit=yearRange&StartYear=2000&EndYear=2005&selRowPerPage=25&Line=33&lstProvince=NS&timeframe=1&Year=2003&Month=9&Day=1#


![screen shot 2017-02-01 at 11 40 15 am](https://cloud.githubusercontent.com/assets/99469/22513652/3b8cd424-e873-11e6-874d-e63d7fa5a6d8.png)

**Comment by richardsc** [comment]
This is a couple years old (but probably newer than our code), and might be relevant:

http://www.fromthebottomoftheheap.net/2015/01/14/harvesting-canadian-climate-data/

**Comment by richardsc** [comment]
Also here's what appears to be an updated version of the above link:

http://www.fromthebottomoftheheap.net/2016/05/24/harvesting-more-canadian-climate-data/

**Comment by dankelley** [comment]
Thanks Clark.  I was going to play a bit with this later today, and this link may save some time.  The main seems to be a change from `climateData` to `climate_data` and from `bulkdata_e` to `hourly_data_e`, etc.

I have to admit that I find it infuriating, how the final operation on the website is a java action and not a link...they make it really difficult to work programmatically. I sure hope DFO does not do such things as this sister agency.

**Comment by richardsc** [comment]
Looks like Gavin even made a package of these scripts (not on CRAN):

https://github.com/gavinsimpson/canadaHCD

**Comment by dankelley** [comment]
Hm, not that hard, after all. The following works (and it will be in the docs for `?met` as soon as I figure out for sure whether to shift the time by 4h, which is what I say in `?met`).
```R
met <- read.met('http://climate.weather.gc.ca/climate_data/bulk_data_e.html?format=csv&stationID=6358&Year=2003&Month=9&timeframe=1&submit=Download+Data')
```
Note: in the previous, I had specified a `Day=` field in this, but I just tried without, and it's fine ... I guess the `timeframe` tells it to give a month of data so it does not need the day.  If they had an API...
